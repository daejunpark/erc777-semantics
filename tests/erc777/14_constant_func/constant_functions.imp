test.preamble();

/*
Functions to test:
    function name() public view returns (string);
    function symbol() public view returns (string);
    function totalSupply() public view returns (uint256);
    function granularity() public view returns (uint256);
    function defaultOperators() public view returns (address[]);
*/
print("name = ");
println(name());
print("symbol = ");
println(symbol());
test.granularity();
test.defaultOperators();
