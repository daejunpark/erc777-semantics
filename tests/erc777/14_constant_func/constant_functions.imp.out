name = RVToken
symbol = RVT
granularity() => 1
defaultOperators() => 8 9 
